src/shared/components/ui/Search/Search.js

```diff
@@ -52,7 +52,7 @@ const Search = ({
     className
   )
 
-  const [isClearActive, setClearActive] = useState(false)
+  const [isClearActive, setClearActive] = useState(defaultValue)
```

Comment:
zrzutuj jeszcze to na boola

Reply:
@nataliapietrzak-landingi Boolean(value) czy !!value ? ;P

Reply:
ja lubię zapis z !!, ale obojętnie